### PR TITLE
babe wake up they cahnged the flag name for SOME reason

### PR DESCRIPTION
**Pull request by redd-rl**
omgggg!!!!!!!!!!!
image

### CODE REVIEW

**Comment by swz-git** [comment]
bump version pls
https://github.com/RLBot/RLBotGUI/blob/8e6c0b39ee26532c0ac298850eae7786ad68bf19/setup.py#L3

**Comment by redd-rl** [comment]
ok pookie done

**Comment by redd-rl** [comment]
THEY REMOVED THE OPTIONS KWARG TOOOOOOOOOOOO OOOOOHHH MY GOOOODDDDDDDDDDDDDDDDDDDDD!!!!! Now Fixed!!!!!
image

File: rlbot_gui/gui.py [hunk 1]
```diff
@@ -713,23 +713,19 @@ def is_full_python():
...
         # and old eel (0.10) defaults nicely to chrome app.
         browser_mode = 'chrome'  # New eel (1.0) needs it to be 'chrome'.
     else:
         browser_mode = 'system-default'
-        options['mode'] = browser_mode
 
     # This disable_cache thing only works if you have tare's fork of eel https://github.com/ChrisKnott/Eel/pull/102
     # installed to pip locally using this technique https://stackoverflow.com/a/49684835
     # The suppress_error=True avoids the error "'options' argument deprecated in v1.0.0", we need to keep the
     # options argument since a lot of our user base has an older version of eel.
```
Comment (azeemba) [inline rlbot_gui/gui.py:726]: If options is not being used anymore, you can delete this comment too

**Review by NicEastvillage — APPROVED** [review]
What a terrible PR

**Comment by redd-rl** [comment]
you're welcome ❤

**Pull request merged**